ikasaneip/platform/ikasan-setup/src/main/resources/providers-properties.xml

```diff
@@ -1114,13 +1114,13 @@
 					   ActionedBy		      		VARCHAR(256) 		NOT NULL,
 					   Action		      			VARCHAR(32) 		NOT NULL,
 					   Event		      			IMAGE 				NOT NULL,
-					   Timestamp		      		NUMERIC(18, 0)       NOT NULL,
+					   Timestamp		      		NUMERIC(18, 0) 		default getdate() NOT NULL,
```

Comment:
I am not sure that 'default getdate()' will work correctly for numeric/ epoch field